decisions/interventions/0001-split-ui-and-service.md

```diff
@@ -0,0 +1,50 @@
+# 1. Split user interface and service by default
+
+Date: 2020-11-02
+
+## Status
+
+Proposed
+
+## Context
+
+The [current HMPPS Digital tech radar][radar] elects Java, Kotlin and node.js as languages in _adopt_. It also
+mentions Ruby (for London); however, the current direction is to reduce regional differences as it is difficult
+to hire developers who are comfortable and happy working with multiple languages.
+
+In short, HMPPS Digital is heading towards using Java, Kotlin and node.js as their primary choices.
+
+The department's talent pool reflects this split; there are:
+
+- many node.js focussed frontend specialists,
+- many Java/Kotlin focussed backend specialists,
+- and a few full-stack developers.
+
+### Team
+
+The current interventions team is mostly on a timed contract with a managed service provider.
+When their contract runs out, we need to be able to backfill developers.
+
+Consequently, we need to choose a technology stack that most other developers in HMPPS already know.
```

Comment:
Thanks, Phil! Since that version, I’ve replaced

> Consequently, we need to choose a technology stack that most other developers in HMPPS already know.

with

> We will **split the user interface** from the API’s component to enforce the use of the business API by default and better utilise the talent we have.

Would you prefer the previous one? I feel the tech stack choice is muted in the new phrasing.

Reply:
📝 Talked to @philroffe offline and all good to go with the new version